docs/podman-rmi.1.md

```diff
@@ -1,9 +1,11 @@
 % podman-rmi(1)
 
 ## NAME
-podman\-rmi - Removes one or more images
+podman\-image\-rm (podman\-rmi) - Removes one or more images
```

Comment:
As this is the "podman rmi" page, I think it should be 
podman\-rmi (podman\-image\-rmi).

Help claims this this is "rmi" not "rm" for `podman image`

Reply:
If you run 
man podman-image-rm
or 
man podman-rmi

You will get this man page. That is what I am trying to reflect.
The name of the file is immaterial.

Reply:
When you display the page, the first line is "podman-rmi(1)"

Then you see:

"podman-image-rmi (podman-rmi)"

I find that very inconsistent.  I'm not focusing on file name here, but what the user sees.

Reply:
Fixed, Now you will see podman-image-rm(1)

Reply:
Ty, that works for me..